onnx/version_converter/adapters/lppool_17_18.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Adapter for MaxPool in default domain from version 8 to 7
+
+#pragma once
+
+#include "onnx/version_converter/adapters/adapter.h"
+#include "onnx/version_converter/adapters/transformers.h"
+
+namespace ONNX_NAMESPACE {
+namespace version_conversion {
+
+class LpPool_17_18 final : public Adapter {
+ public:
+  explicit LpPool_17_18() : Adapter("LpPool", OpSetID(17), OpSetID(18)) {}
+
+  void adapt_lppool_17_18(std::shared_ptr<Graph>, Node* node) const {
```

Comment:
I guess this file is not needed? `convert.h` seems to use a CompatibleAdapter, which seems correct.

Reply:
Good catch, I deleted the file and used `registerAdapter("LpPool", 17, 18, SetAttribute(kceil_mode, 0));` instead.